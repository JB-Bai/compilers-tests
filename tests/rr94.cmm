int;
float OYg3Odtum;
struct G73vMc3{
}
hx9orFM[2934];

