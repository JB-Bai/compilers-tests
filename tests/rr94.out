Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      SEMI
    ExtDefList (2)
      ExtDef (2)
        Specifier (2)
          TYPE: float
        ExtDecList (2)
          VarDec (2)
            ID: OYg3Odtum
        SEMI
      ExtDefList (3)
        ExtDef (3)
          Specifier (3)
            StructSpecifier (3)
              STRUCT
              OptTag (3)
                ID: G73vMc3
              LC
              RC
          ExtDecList (5)
            VarDec (5)
              VarDec (5)
                ID: hx9orFM
              LB
              INT: 2934
              RB
          SEMI
